lib/Chat/MessageParser.php

```diff
@@ -103,4 +107,30 @@ protected function setActor(Message $message): void {
 			$displayName
 		);
 	}
+
+	public function isSharedFile(string $message): bool {
+		$parameters = $this->getParametersFromMessage($message);
+		return !empty($parameters['share']);
+	}
+
+	private function getParametersFromMessage(string $message): array {
```

Comment:
```suggestion
	protected function getParametersFromMessage(string $message): array {
```

Reply:
We use protected basically everywhere in talk to make it easier to extend the code and inherit it.

> $ find lib/ -iname "*.php" | xargs grep "protected function" | wc -l
> 247
> $ find lib/ -iname "*.php" | xargs grep "private function" | wc -l
> 57

so unless really necessary (and it's not here) please use protected and never use private

Reply:
This is a good command to list the files that could be outside this pattern and a good start point to define a rule, put in the linter and create a GitHub issue to check if all private methods are really necessary to be private.